lib/advanced-open-file-view.coffee

```diff
@@ -323,14 +335,18 @@ class AdvancedFileView extends View
     @getFileList (files) -> @renderAutocompleteList files
 
   suggestPath: ->
-    activePath = atom.workspace.getActiveTextEditor()?.getPath()
-    if activePath
-      activeDir = path.dirname(activePath) + path.sep
-      @miniEditor.setText activeDir
-    else
-      projectPaths = atom.project.getPaths()
-      if projectPaths.length > 0
-        @miniEditor.setText projectPaths[0] + path.sep
+    suggestedPath = ''
+    switch atom.config.get("advanced-open-file.defaultInputValue")
+      when DEFAULT_ACTIVE_FILE_DIR
+        activePath = atom.workspace.getActiveTextEditor()?.getPath()
+        if activePath
+          suggestedPath = path.dirname(activePath) + path.sep
+      when DEFAULT_PROJECT_ROOT
+        projectPaths = atom.project.getPaths()
+        if projectPaths.length > 0
+          suggestedPath = projectPaths[0] + path.sep
```

Comment:
Do you need to include a case here for DEFAULT_EMPTY, or is the goal to rely on the implicit "do nothing" behavior?